src/app/login/login.service.ts

```diff
@@ -0,0 +1,25 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+
+import { environment } from 'src/environments/environment';
+import { User } from './login.model';
+
+const userUrl = `${environment.apiUrl}/users`;
+const loginUrl = `${environment.apiUrl}/login`;
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoginService {
+
+  constructor(private http: HttpClient) { }
+
+  addUser(user: User): Observable<unknown> {
```

Comment:
return type unknown?

Reply:
As of now, I have not decided what to return. Once it is decided, I will update it.